README.md

```diff
@@ -435,6 +438,33 @@ dns:
   - Description: Histogram of response times for DNS checks
   - Labelled with `target`
 
+### Check: Traceroute
```

Comment:
I think we should write something about the capabilities needed for the check to succeed and that the user can still use the sparrow without these capabilities for the other checks.